.travis.yml

```diff
@@ -59,9 +59,10 @@ jobs:
               - docker run -u $UID --privileged -t --rm
                 --volume=/etc/passwd:/etc/passwd:ro
                 --volume=$PWD:/hlwm:rw
+                --volume=$HOME/.ccache-eoan-clang:/.ccache:rw
                 --volume=$HOME/.tox-cache:/hlwm/.tox:rw
                 focal sh -c '
-                    /hlwm/ci/build.py --build-dir=/hlwm/build-clang-10 --cxx=clang++-10 --cc=clang-10 --build-type=Debug --check-using-std --iwyu --clang-tidy --flake8
+                    /hlwm/ci/build.py --build-dir=/hlwm/build-clang-9 --cxx=/hlwm/ci/clang++-and-tidy.sh --cc=/hlwm/ci/clang-and-tidy.sh --build-type=Debug --ccache=/.ccache --compile --check-using-std --iwyu --flake8
```

Comment:
Merge error? (build-clang-9)

Reply:
Yup! It doesn't really matter. I took this opportunity to change both build dirs to `/hlwm/build`, which is totally sufficient after this change.

Reply:
Thanks!